analysis/deep2_r23_o32/archives/refactor_into_msc/dust_attenuation_functions.py

```diff
@@ -55,12 +55,6 @@ def call_cardelli(lam0): #, extrapolate=False):
     return k_values
 
 
-'''k_ascii = fitspath_ini+'/cardelli_k_values.tbl'
-    n3 = ('Line_Name','K_value')
-    tab3 = Table([line_name, k_values])
-    asc.write(tab3, k_ascii, format = 'fixed_width_two_line')'''
-
-
 
 
```

Comment:
Delete these lines as well so there aren't that many lines after return k_values.

Reply:
I created #64 for this issue. Will address outside this PR.